src/main/webapp/map.html

```diff
@@ -0,0 +1,59 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Google Maps</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAIJxmuluUxbW2r2rZgOebw7B0U1rRtrvA"></script>
+    <script>
+      /** Creates a map that shows landmarks around Google. */
```

Comment:
I think it might be worth creating a new file for all the javascript and then adding that file in through the script tag, in case we want to use the js code somewhere else on the site and also for some organization

https://github.com/Ivyzhang1998/team-16-codeu/blob/9514802966d2c260d9041072235ab91fccb69f7d/src/main/webapp/map.html#L7-L37

Reply:
This is a great suggestion!